firelynetsdk/index.rst

```diff
@@ -14,18 +18,17 @@ The library provides:
 - Validator to validate instances against profiles
 - A lightweight in-memory terminology server
 
-On these pages we provide you with the documentation you need to get up and running with the API.
-We'll first explain how the FHIR model is represented in the API and give you code examples to work with the model.
+On these pages we provide you with the documentation you need to get up and running with the SDK.
+We'll first explain how the FHIR model is represented in the SDK and give you code examples to work with the model.
 The FhirClient and its methods will also be demonstrated. Within an hour you can create your own simple
 FHIR client!
 
-After those topics to get you started, we have added some pages that delve deeper into nice API features, such
+After those topics to get you started, we have added some pages that delve deeper into nice SDK features, such
 as parsing and serializing FHIR data, working with transactions, and using the ResourceIdentity functionality.
 
-.. note:: All code examples on these pages are for the STU3 version of the library. Some of them can still be used
-	with the DSTU2 library, but you might need to tweak the code a bit.
+.. note:: All code examples on these pages are for the STU3 version of the library. 
 
-Please look at the :ref:`api-contact` page for ways to ask questions, contribute to the API, or reach out to other
+Please look at the :ref:`api-contact` page for ways to ask questions, contribute to the SDK, or reach out to other
```

Comment:
should the 'api-contact' also be page also be renamed to 'sdk-contact' ?

Reply:
hmm, maybe you are right. Because these are meta labels I left them unchanged. Also like the name of the pictures, they also have the prefix api. But I will change it.